client/webserver/site/src/js/markets.js

```diff
@@ -246,6 +246,31 @@ export default class MarketsPage extends BasePage {
       page.cancelPass.value = ''
     }
 
+    // Resize elements on screen when the page is resized
+    this.resize = () => {
+      let marketTableHeight
+      const largeScreenCutoffWidth = 992
+      if (window.innerWidth < largeScreenCutoffWidth) {
+        // In smaller screens, the order panel and the market tables are stacked
+        // on top of each other, and we want the market tables to have the same height
+        // as the live orders table.
+        marketTableHeight = page.liveTableContainer.offsetHeight - page.liveTableHead.offsetHeight
```

Comment:
The `orders-left col-24` div seems alright when narrow, but it does seem like the computation over allocates height a bit.  See the sell orders table below:

![image](https://user-images.githubusercontent.com/9373513/128397865-5ca84da9-d19e-41e2-a87f-4b6f0385056e.png)

Compared with a testnet setup (no vscroll, but also no more height than needed):

![image](https://user-images.githubusercontent.com/9373513/128399953-ab3723c4-487f-4149-af6a-e0b0b1c207f9.png)